compositor_api/src/types/from_register_output.rs

```diff
@@ -161,38 +170,45 @@ impl TryFrom<WhipOutput> for pipeline::RegisterOutputOptions<output::OutputOptio
             ));
         }
 
+        if video.as_ref().filter(|v| v.encoder.is_some()).is_some() {
```

Comment:
filter on option is wierd, you could `video.and_then(|v| v.encoder).is_some()`

but personally I would prefer something like this:
```
if let Some(OutputWhipVideoOptions{ encoder: Some(encoder), .. })) = &video {
```